### PR TITLE
[Draft] Add second instance support

### PR DESCRIPTION
**Pull request by NickTikhonov**
Adds second instance support according to [these instructions.](https://github.com/asticode/go-astilectron/issues/284#issuecomment-712233422)

TODO: test the integration.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 const { app } = require("electron");
 const { start, getLastWindow } = require("./index");
+const client = require('./src/client.js')
+const consts = require('./src/consts.js')
```
Comment (asticode) [inline main.js]: I know this is only a draft, but you should be able to get `client` and `consts` like this `const { start, getLastWindow, client, consts } = require("./index");` if you export them at the bottom of `index.js`

**Comment by NickTikhonov** [comment]
@asticode sure thing - happy to make that refactor once I get the integration working. By the way, what's the recommended way to test a change like this? Will I need to change the bundler also?

**Comment by asticode** [comment]
> By the way, what's the recommended way to test a change like this?

I usually modify the [demo](https://github.com/asticode/go-astilectron-demo) just for this occasion (in your case simply add logs on the `second instance` event), run the `bundler` with the `-a` flag (if you run `astilectron-bundler -a /path/to/your/local/astilectron/dir` it will bind your local source files to your binary), run the binary and checks whether the new is working and the old stuff is still working.

> Will I need to change the bundler also?

`go.mod` will need to be bumped in all repos but I'll take care of it once the PR is merged.

**Comment by Joe-Improbable** [comment]
I tested this PR, which works fine. 

I just pushed a commit that cleans up the JS imports. 

Let me know what you think

**Comment by asticode** [comment]
FYI I've created a `v0.42.0` tag

Cheers

**Pull request merged**